applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/Kconfig.defaults

```diff
@@ -80,13 +80,20 @@ config BT_DEVICE_APPEARANCE
 config BT_GAP_PERIPHERAL_PREF_PARAMS
 	default n
 
+config BT_GATT_AUTO_SEC_REQ
+	default n
+
 config BT_VCP_VOL_REND
 	default y
 
 config BT_MCC
 	default n if WALKIE_TALKIE_DEMO
 	default y
 
+config BT_PAC_SNK_NOTIFIABLE
+	# For fixing compaitablility issue with Android 14
```

Comment:
Spell check
```suggestion
	# For fixing compatibility issue with Android 14
```

Reply:
Hmmm, time to check why dictionary not work.